playwright-e2e/pages/page-base.ts

```diff
@@ -92,6 +92,7 @@ export default abstract class PageBase implements PageInterface {
 
   /** Click "Agree" button */
   async agree(): Promise<void> {
+    await expect(this.getIAgreeButton()).toBeEnabled();
```

Comment:
Attempt to address a flaky issue. Briefly, here is what happens:

- Fill out Signature input
- `I Agree` button is enabled
- Clicked
- Button becomes disabled
- Quickly (few seconds), button is again re-enabled
- Page remains unchanged

For example, see this [CI job test results](https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/20805/workflows/9a5b5dcd-5849-4062-b443-06f29d4eed64/jobs/31200/artifacts).

Reply:
If it helps (and if I'm following right) - that one is a really old flaky issue - consent related submit buttons (for automated tests) occasionally but not consistently seem to need to be double clicked

Reply:
Good info. Thank you Kiara. 
@alextanlan Can this UI (code) issue gets looked into to avoid testing headaches?